### DISCUSSION TITLE
NuxtPage  as a root element

### DISCUSSION THREAD

**Discussion by Nugine** [nuxt/nuxt#19596] [Questions]
The [guide](https://nuxt.com/docs/guide/directory-structure/app#usage-with-pages) says

> Since Nuxt 3 uses `<Suspense>` inside `<NuxtPage>`, it cannot be set as a root element.

What does it mean? Can I write `app.vue` like this?
```vue
<template>
  <NuxtPage/>
</template>
```

Disclaimer: I'm new to `vue` and `nuxt`.

**Comment by oliver139** [comment]
New to Nuxt and I want to know about this as well..

**Comment by oliver139** [comment]
As per the sample code from https://nuxt.com/docs/api/components/nuxt-page
I think it is ok to do so?
